components/UploadedItems.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+
+var items = 50;
```

Comment:
Do we really need this variable?

Reply:
Initially, I thought that I have to show the names of the books uploaded but then I realize it's only the no. the books. I'll fix that